app/javascript/test/__snapshots__/orderManagerFilesetsForm.spec.js.snap

```diff
@@ -20,12 +20,14 @@ exports[`OrderManagerFilesetsForm.vue has the expected html structure 1`] = `
       id="unitLabel"
       label="Label"
       placeholder="e.g., p."
+      value="p. "
```

Comment:
These updates seem to be sensible to me, as `v-model` binds to the `unitLabel` property, and hence, the default value populates this element